spec/unit/rack/context_filter_spec.rb

```diff
@@ -38,7 +38,7 @@ def env_for(url, opts = {})
       { 'HTTP_USER_AGENT' => 'Bingo Agent' }
     end
 
-    it "adds User-Agent to the context" do
+    it "adds UserAgent to the context" do
```

Comment:
Could you please revert this? It's not a typo. https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent